### PR TITLE
[#4006] anonymous uuid for mixpanel

### LINKED ISSUE #4006: Unique anonymous mixpanel id's
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a developer, I want to mixpanel ID's to be anonymous so that privacy is preserved.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Improvement

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: Mixpanel distinct id's are currently a hash of the wallet pub key, but the id should be truly anonymous and not be linked to the user at all.

#### Expected behavior
Mixpanel distinct id is anonymous and non-traceable

#### Actual behavior
Mixpanel distinct id is a hash of the wallet

### Solution

Possible solution for iOS:

> In any event, the best you can now do for identification purposes is to use a UUID (Universally Unique IDentifier). This has to be on a per-app basis. That is, there is no way to identify the device any longer, but you can identify an app on a device.As long as the user doesn't completely delete the app, then this identifier will persist between app launches, and at least let you identify the same user using a particular app on a device.

### PR DESCRIPTION
**Pull request by rasom**
fixes #4006

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/utils/handlers.cljs [hunk 1]
```diff
@@ -100,7 +100,7 @@
          (let [event    (get-coeffect context :event)
                offline? (or (= :offline (:network-status new-db))
                             (= :offline (:sync-state new-db)))
-               anon-id  (ethereum/sha3 current-account-id)]
```
Comment (jeluard) [inline src/status_im/utils/handlers.cljs:103]: `ethereum` ns declaration can be removed. Same for `current-account-id` definition.

**Review by jeluard — CHANGES_REQUESTED** [review]

File: src/status_im/utils/handlers.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/utils/handlers.cljs:103]: done

**Review by jeluard — APPROVED** [review]

**Comment by asemiankevich** [comment]
image

**Comment by chadyj** [comment]
@asemiankevich I just noticed that the screenshot above is missing distinct id. That isn't being sent to Mixpanel.

**Pull request merged**